packages/shared/src/hooks/userCompany/useUserCompaniesQuery.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query';
+import { UserCompany } from '../../lib/userCompany';
+import { GET_USER_COMPANIES } from '../../graphql/users';
+import { generateQueryKey, RequestKey, StaleTime } from '../../lib/query';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { gqlClient } from '../../graphql/common';
+
+interface UseUserCompaniesQuery {
+  userCompanies: UserCompany[];
+  isVerified: boolean;
+  isLoading: boolean;
+}
+export const useUserCompaniesQuery = (): UseUserCompaniesQuery => {
```

Comment:
No `useVerified`? 😢

Reply:
I thought about it, but then didn't really saw a clear case for it 😅 (as we need the main query anyway)